exercise-2/exercise-2-7/exercise-2-7.js

```diff
@@ -0,0 +1,16 @@
+function createTable() {
+  // Input table rows and column
+  let rowNumber = window.prompt('Input number of rows', '1');
+  let colNumber = window.prompt('Input number of columns', '1');
+
+  // Handle insert row
+  for (let row = 0; row < parseInt(rowNumber); row++) {
```

Comment:
Should not use parseInt in the looping. Because it's will make process slower when each loop will try parseInt. 
Using a variable instead

```
const totalRow = parseInt(rowNumber);
for (let row = 0; row < totalRow; row++) {
...
}
```